cylc/flow/xtriggers/suite_state.py

```diff
@@ -58,9 +58,6 @@ def suite_state(suite, task, point, offset=None, status='succeeded',
                different location to what is specified in the global
                configuration (usually ``~/cylc-run``).
 
-        debug (bool):
-            Flag to enable debug information.
-
```

Comment:
Can you remove `(except ''debug")` from the bottom of the docstring too? (line 71)

Reply:
Done!